crates/sui-network/build.rs

```diff
@@ -155,6 +155,15 @@ fn build_anemo_services(out_dir: &Path) {
                 .codec_path(codec_path)
                 .build(),
         )
+        .method(
+            anemo_build::manual::Method::builder()
+                .name("get_peer_latest_checkpoint_info")
```

Comment:
suggestion-I think "peer" is not needed as part of the name of this method/associated types, all RPCs are implicitly requested from a peer so GetLatestCheckpointInfo would be simpler

I'm also not quite clear on the function of "latest" in this name - is earlier checkpoint info available through some other means (or will become so eventually)? If not, then it seems something like `GetCheckpointInfo` or `GetCheckpointWatermarks` says everything you need to about what this method does

(I might have a slight preference for something like "Watermarks" or "Availability", because "Info" is a little vague and sounds like it could be part of the checkpoint data itself rather than metadata about which checkpoints are available on this node)

Reply:
will rename to `GetCheckpointAvailability`